src/vinylcontrol/vinylcontrolxwax.cpp

```diff
@@ -836,7 +836,7 @@ float VinylControlXwax::getAngle() {
         return -1.0;
     }
 
-    float rps = timecoder_revs_per_sec(&timecoder);
+    double rps = timecoder_revs_per_sec(&timecoder);
```

Comment:
I think float was correct here, in the calculation below the f suffix is missing.